processor/src/operations/mod.rs

```diff
@@ -25,6 +26,15 @@ impl Process {
         &mut self,
         op: Operation,
         host: &mut impl Host,
+    ) -> Result<(), ExecutionError> {
+        self.execute_op_with_error_ctx(op, host, &ErrorContext::default())
```

Comment:
Using `ErrorContext::default()` here would mean that we can't do any source mappings, right? Or am I missing something?

Reply:
Ah yes, this method currently only being used by
1. all tests that want to check the correctness of operation execution (>100)
2. "inner calls" to `execute_op()` as discussed in #1538. Since these call either `Noop` or `Drop`, which cannot fail, they don't need to propagate the error context.

If we address #1538, then we could rename `execute_op()` and make it test-only for convenience. Another one of my goals in leaving it untouched was to not blowup the diff in this PR.